tests/testthat/test-robustbayesianmetaanalysis.R

```diff
@@ -133,125 +133,125 @@ fitted_path <- file.path("robmaFit.RDS")
   })
   
   test_that("Model Summary table results match", {
-    table <- results[["results"]][["model_preview"]][["collection"]][["model_preview_overall_summary"]][["data"]]
+    table <- results[["results"]][["modelPreview"]][["collection"]][["modelPreview_overallSummary"]][["data"]]
```

Comment:
modelPreviewOverallSummary?

Reply:
these are automatically generated names by jaspTools, can't do much about them - it's linking the name of a container `modelPreview` with the name of a table inside the container `overallSummary`

Reply:
Okay no worries.